docs/node/run-your-node/archive-node.md

```diff
@@ -31,9 +32,42 @@ mode: archive
 
 :::info
 
-Keep other settings unchanged from the full-node. For example in order to
-serve archived runtime state, the node needs to have the runtime configured
-and have the state present.
+Keep all other settings the same as those for a full node. For example, to serve archived runtime
```

Comment:
I would rather have a complete Client config pasted above with the archive mode enabled, so one can simply set up their own archive node from scratch when reading the chapter. You don't need SGX, a configured entity or being elected in the validator for running the archive node right?

Reply:
Ok, i updated the configs. Need to double check if the configs are correct, but I believe these should work, since i picked them from docs git history.